src/apps/loyalty/server/index.tsx

```diff
@@ -0,0 +1,46 @@
+import * as express from "express"
+import * as path from "path"
+import * as React from "react"
+import { renderToString } from "react-dom/server"
+import * as styleSheet from "styled-components/lib/models/StyleSheet"
+import renderPage from "./template"
+
+import Inquiries from "../containers/inquiries"
+import Login from "../containers/login"
+
+const app = express()
+
+app.use(express.static(path.resolve(__dirname)))
+
+app.get("/", (req, res) => {
+  // check if the user is logged in, if not redirect to /login
```

Comment:
It’s not clear to me what this comment is doing here if the actual check is performed in `/inquiries` ?